tests/includes/juju.sh

```diff
@@ -281,6 +281,11 @@ juju_bootstrap() {
 	bash -c "exec ${command} $*" 2>&1 | OUTPUT "${output}"
 	echo "${name}" >>"${TEST_DIR}/jujus"
 
+	if [[ ${BOOTSTRAP_PROVIDER} != "k8s" ]]; then
+		juju switch "${name}:controller"
+		wait_for_machine_agent_status "0" "started"
```

Comment:
Was ist das?

Reply:
we don't have machines in k8's. It was what we discussed in standup a while ago.

Reply:
OK, I don't remember the conversation. But this seems OK, more was wondering why we needed this, and there was no comment.